src/components/Search/index.jsx

```diff
@@ -0,0 +1,89 @@
+import algoliasearch from 'algoliasearch/lite';
+import { default as React, useState, useMemo } from 'react';
+import { InstantSearch } from 'react-instantsearch-dom';
+import {
+  ThemeProvider,
+  makeStyles,
+  Box,
+  List,
+  ClickAwayListener,
+} from '@material-ui/core';
+import SearchBox from './search-box';
+import SearchResult from './search-result';
+
+//{window.addEventListener('resize', handleClose)}
+
+const useStyles = makeStyles({
+  root: {
+    background: 'white',
+  },
+  popover: {
+    background: 'white',
+    position: 'absolute',
+    zIndex: 2,
+  },
+  resultsBorder: {
+    borderRadius: 3,
+    borderColor: '#D3D3D3',
+  },
+  search: {
+    maxHeight: 750,
+    width: 'auto',
+    height: 'auto',
+    boxShadow: 5,
+    overflow: 'auto',
+    overflowX: 'hidden',
+  },
+});
+
+export default function Search({ indices }) {
+  const [query, setQuery] = useState();
+  const searchClient = useMemo(
+    () =>
+      algoliasearch(
+        process.env.GATSBY_ALGOLIA_APP_ID,
+        process.env.GATSBY_ALGOLIA_SEARCH_KEY
+      ),
+    []
+  );
+  const theme = useStyles();
```

Comment:
The convention here is to name this `classes`, not `theme`, since it's the generated CSS class names for the styles (eg. in `classes.root`. `theme` is actually a different concept in material-ui, referring to the styling context across all components.